src/main/java/com/matt/forgehax/mods/commands/HelpCommand.java

```diff
@@ -220,4 +220,14 @@ public Command respawn(CommandBuilders builder) {
             })
         .build();
   }
+
+@RegisterCommand
+    public Command clearChat(CommandBuilders builders) {
+      return builders
+        .newCommandBuilder()
+        .name("clear")
+        .description("Clears chat")
+        .processor(d -> MC.ingameGUI.getChatGUI().clearChatMessages(true))
```

Comment:
This should be run inside MC.addScheduledTask to prevent a possible and rare concurrent modification crash